.storybook/stories/Particles.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react'
+import { Vector3 } from 'three'
+
+import { Setup } from '../Setup'
+
+import { Particles, Environment, ContactShadows, PerspectiveCamera, OrbitControls } from '../../src'
+
+export default {
+  title: 'Staging/Particles',
+  component: Particles,
+  decorators: [
+    (storyFn) => (
+      <Setup cameraPosition={new Vector3(1, 1, 1)} controls={false}>
+        {storyFn()}
+      </Setup>
+    ),
+  ],
+}
+
+export const ParticlesStory = (props) => (
+  <>
+    <Particles {...props} color="cyan" />
```

Comment:
Is there no way to further customize the particles? eg. number, movement, what gets rendered?